src/nft-card.ts

```diff
@@ -182,6 +197,13 @@ export class NftCard extends LitElement {
               .horizontal=${this.horizontal}
               @new-event="${this.eventHandler}"
               .asset=${this.asset}
+              .state=${({
```

Comment:
future improvement idea (not now): for passing around global state, i ran into https://github.com/matthewp/haunted